MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 include requirements.txt
+include hta/configs/logging.config
+include hta/configs/trace_analyzer.json
 include docs/conf.py
```

Comment:
Should these be sorted alphabetically?

Reply:
i'll make the edits using the skip ci flag so that circle ci is not triggered for these changes